core/src/test/scala/com/karumi/shot/reports/ConsoleReporterSpec.scala

```diff
@@ -0,0 +1,141 @@
+package com.karumi.shot.reports
+
+import com.karumi.shot.base64.Base64Encoder
+import com.karumi.shot.domain.{
+  DifferentScreenshots,
+  Screenshot,
+  ScreenshotNotFound,
+  ScreenshotsComparisionResult
+}
+import com.karumi.shot.ui.Console
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
+
+object ShotErrorMessages {
+  val testBrokenMessage =
+    "❌  Hummmm...the following screenshot tests are broken:\n"
+  val screenShotNotFoundMessage =
+    "   🔎  Recorded screenshot not found for test: name"
+  val base64EnabledMessage =
+    "\n🤖  The option printBase64 is enabled. In order to see the generated diff images, run the following commands in your terminal:"
+  val base64GenerationErrorMessage =
+    "\t❌ Base64 image generation error, image source not found."
+  val base64MessageToCopy = "\t> echo 'base64' | base64 -D > name"
+  val diffScreenshotsModifiedMessage =
+    "   🤔  The application UI has been modified for test: name"
+  val diffOriginalScreenshotMessage =
+    "            💾  You can find the original screenshot here: recordedPath"
+  val diffRecordedScreenshotMessage =
+    "            🆕  You can find the new recorded screenshot here: temporalPath/name"
+}
+
+class ConsoleReporterSpec
```

Comment:
I can't stop but feeling this is an over-specification, the most clear hint being that we are duplicating strings. Don't get me wrong, I think we should keep these but maybe we should replace the mocking library by one that doesn't verify every single call to its mocks.

Reply:
I know @Serchinastico but in this case, I want to know the messages are showing for base64 and not showing when if it's appropriate. The library force me to add every single message, but in any case to me is not an over-specification, I think there is important to test that the base64 it is showing or not.